src/page/summary-step.page.ts

```diff
@@ -4,7 +4,8 @@ export class SummaryStepPage {
   private proceedCheckoutBtn: ElementFinder;
 
   constructor () {
-    this.proceedCheckoutBtn = $('.cart_navigation span');
+    // it is more meaningful if we use also an 'a'
+    this.proceedCheckoutBtn = $('.cart_navigation a span');
```

Comment:
I think that `.cart_navigation span` is the only combination in that page that points to the proceed to checkout button. Maybe, the a is redundant